### PR TITLE
I173 add delete buttons

### LINKED ISSUE #173: Add Delete Buttons
## Basic Information
Add delete button to the edit modals for tournament, venues and institutions

### PR DESCRIPTION
**Pull request by domcain**
## Change Summary
**[Briefly explain the changes that you made. Just high-level stuff]**
- delete button functionality added to tournaments, venues and institutions
- delete button adaptive positioning fixed
- deletion notification 

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [X] The pull request title has an issue number
- [X] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related issue

- Resolve #173

### CODE REVIEW

File: pages/admin/institutions.vue [hunk 1]
```diff
@@ -108,12 +112,19 @@ const handleEdit = (row) => {
...
+            type="Submit"
+            class="text-red-700"
+            @click="deleteInstitution(formInput.id)"
+          />
           <Button
             button-text="Update"
             button-color="bg-gold"
             type="Submit"
-            class="m-5 ml-8"
+            class=""
```
Comment (rezbyte) [inline pages/admin/institutions.vue:127]:
```suggestion
```

File: stores/institutions.js [hunk 2]
```diff
@@ -304,5 +304,11 @@ export const useInstitutionStore = defineStore("institution", {
         console.log("NO TEAMS", e);
       }
     },
+    async deleteInstitution(id) {
```
Comment (rezbyte) [inline stores/institutions.js:307]:
```suggestion
    deleteInstitution(id) {
```
This doesn't use `await` so no need to make it async.

File: stores/tournaments.js [hunk 3]
```diff
@@ -75,5 +75,11 @@ export const useTournamentStore = defineStore("tournament", {
         }
       });
     },
+    async deleteTournament(id) {
```
Comment (rezbyte) [inline stores/tournaments.js:78]:
```suggestion
    deleteTournament(id) {
```
`async` not needed.

File: stores/venues.js [hunk 4]
```diff
@@ -40,5 +40,11 @@ export const useVenueStore = defineStore("venue", {
         this.venues.push({ id: doc.id, ...doc.data() });
       });
     },
+    async deleteVenue(id) {
```
Comment (rezbyte) [inline stores/venues.js:43]:
```suggestion
   deleteVenue(id) {
```

File: pages/admin/venues.vue [hunk 5]
```diff
@@ -142,12 +146,19 @@ const headers = [
...
+            type="Submit"
+            class="text-red-700"
+            @click="deleteVenue(formInput.id)"
+          />
           <Button
             button-text="Update"
             button-color="bg-gold"
             type="Submit"
-            class="m-5 ml-8"
+            class=""
```
Comment (rezbyte) [inline pages/admin/venues.vue:161]:
```suggestion
```

File: pages/admin/tournaments.vue [hunk 6]
```diff
@@ -42,12 +42,19 @@
...
+            type="Submit"
+            class="text-red-700"
+            @click="deleteTournament(form.id)"
+          />
           <Button
             button-text="Update"
             button-color="bg-gold"
             type="Submit"
-            class="m-5 ml-8"
+            class=""
```
Comment (rezbyte) [inline pages/admin/tournaments.vue:57]:
```suggestion
```

**Review by rezbyte — CHANGES_REQUESTED** [review]
Looks good Dom!
Found some typos to fix.
It might also be worth adding notifications.

**Comment by domcain** [comment]
@rezbyte  I have added notifications upon successful deletion of venues, institutions and tournaments

**Comment by rezbyte** [comment]
> @rezbyte I have added notifications upon successful deletion of venues, institutions and tournaments. Will fix the merge conflicts, then make another PR

Nice job!
Although there's no need to write your own code to control the notifications.
The `useNotification` composable implements everything for you.
See `pages/admin/teams.vue` for a usage example.
I'm happy to help if you'd like.

**Review by rezbyte — DISMISSED** [review]
Great job, Dom! 🎆 
As discussed, it'll be refactored to use `useNotification` in another PR.

**Review by rezbyte — APPROVED** [review]
I merged `development` into this branch, just re-approving it to please GitHub.

**Pull request merged**